pservice/lib/libpdo_enclave/secret_enclave.cpp

```diff
@@ -680,6 +678,7 @@ pdo_err_t VerifyEnclaveInfo(const std::string& enclaveInfo,
 
 
     return result;
+
```

Comment:
revert these. the formatting is not improved and the only changes i see are formatting.

Reply:
thank you, done